docs/make.jl

```diff
@@ -0,0 +1,203 @@
+        "Training",
+        [
+            MultiDocumenter.Column(
+                "Data Wrangling",
+                [
+                    MultiDocumenter.MultiDocRef(
+                        upstream = joinpath(clonedir, "MLUtils"),
+                        path = "mlutils",
+                        name = "MLUtils",
+                        giturl = "https://github.com/JuliaML/MLUtils.jl.git",
+                    ),
+                    MultiDocumenter.MultiDocRef(
+                        upstream = joinpath(clonedir, "OneHotArrays"),
+                        path = "onehotarrays",
+                        name = "OneHotArrays",
+                        giturl = "https://github.com/FluxML/OneHotArrays.jl.git",
+                    ),
+                ]
+            ),
+            MultiDocumenter.Column(
+                "Data Augmentation",
+                [
+                    MultiDocumenter.MultiDocRef(
+                        upstream = joinpath(clonedir, "DataAugmentation"),
+                        path = "dataaugmentation",
+                        name = "DataAugmentation",
+                        giturl = "https://github.com/FluxML/DataAugmentation.jl.git",
+                    ),
+                ],
+            ),
+            MultiDocumenter.Column(
+                "Datasets",
+                [
+                    MultiDocumenter.MultiDocRef(
+                        upstream = joinpath(clonedir, "MLDatasets"),
+                        path = "mldatasets",
+                        name = "MLDatasets",
+                        giturl = "https://github.com/JuliaML/MLDatasets.jl.git",
+                    ),
+                ],
+            ),
+            MultiDocumenter.Column(
+                "Schedulers",
+                [
+                    MultiDocumenter.MultiDocRef(
+                        upstream = joinpath(clonedir, "ParameterSchedulers"),
+                        path = "paramschedulers",
+                        name = "ParameterSchedulers",
+                        giturl = "https://github.com/FluxML/ParameterSchedulers.jl.git",
+                    ),
+                ],
+            ),
+            # MultiDocumenter.Column(
+            #     "High-level training libraries",
+            #     [
+            #         MultiDocumenter.MultiDocRef(
+            #             upstream = joinpath(clonedir, "FluxTraining"),
+            #             path = "fluxtraining",
+            #             name = "FluxTraining",
+            #             giturl = "https://github.com/FluxML/FluxTraining.jl.git",
+            #         ),
+            #         MultiDocumenter.MultiDocRef(
+            #             upstream = joinpath(clonedir, "FastAI"),
+            #             path = "fastai",
+            #             name = "FastAI",
+            #             giturl = "https://github.com/FluxML/FastAI.jl.git",
+            #         ),
+            #     ],
+            # ),
```

Comment:
These are commented out because FastAI and FluxTraining use Pollen, which does not load very well at all and also takes up a massive amount of space on disk. Long term we might just have to move these to Documenter.jl for better integration